### PR TITLE
added yaehmop recipe

### PR DESCRIPTION
**Pull request by richardjgowers**

### CODE REVIEW

**Comment by richardjgowers** [comment]
@micaela-matta are you happy to be a maintainer of this?

**Comment by richardjgowers** [comment]
@conda-forge/help-c-cpp this is ready for review

**Comment by micaela-matta** [comment]
@richardjgowers  sure.

File: recipes/yaehmop/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cd $SRC_DIR/tightbind
+make
+
+mv bind yaehmop
+cp yaehmop $PREFIX
```
Comment (beckermr) [inline recipes/yaehmop/build.sh]: Executables should be copied to `$PREFIX/bin`. Is this an executable?

File: recipes/yaehmop/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  commands:
+    - yaehmop -v
+
+about:
+  home: https://github.com/greglandrum/yaehmop
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Yet Another extended Hueckel Molecular Orbital Package'
+
```
Comment (beckermr) [inline recipes/yaehmop/meta.yaml]: Let's remove this space.

File: recipes/yaehmop/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{% set name = "yaehmop" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/greglandrum/yaehmop/archive/master.tar.gz
```
Comment (beckermr) [inline recipes/yaehmop/meta.yaml]: Please don't pull from master. A release on github should be tagged and then the package should pull the release tarball.

File: recipes/yaehmop/LICENSE.txt [hunk 4]
```diff
@@ -0,0 +1,28 @@
+Copyright (C) 1995 Greg Landrum
```
Comment (beckermr) [inline recipes/yaehmop/LICENSE.txt]: The license is in downloaded source already, so you do not need to include it here.

Comment (beckermr) [reply recipes/yaehmop/LICENSE.txt]: Thus you should delete this file.

File: recipes/yaehmop/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  commands:
+    - yaehmop -v
+
+about:
+  home: https://github.com/greglandrum/yaehmop
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```
Comment (beckermr) [inline recipes/yaehmop/meta.yaml]: In the downloaded source, the license is called `license.txt` (notice it is lowercase). Change this line to lowercase so conda pulls the license from the download.

**Comment by richardjgowers** [comment]
@beckermr thanks for the review.  The Makefile for this project (from the 90s) doesn't allow a custom `--prefix` option, so I just did a manual make install

File: recipes/yaehmop/meta.yaml (hunk 3 above)
Comment (richardjgowers) [reply recipes/yaehmop/meta.yaml]: I've poked the person upstream about making a github release we can target

File: recipes/yaehmop/build.sh (hunk 1 above)
Comment (richardjgowers) [reply recipes/yaehmop/build.sh]: Yep, fixed

File: recipes/yaehmop/build.sh [hunk 6]
```diff
@@ -1,5 +1,5 @@
 cd $SRC_DIR/tightbind
 make
-
+# manual make install essentially..
```
[the remark this answers is not in the document]
Comment (beckermr) [reply recipes/yaehmop/build.sh]: Comment on lack of prefix option should go here!

**Comment by richardjgowers** [comment]
@beckermr I think we're all good to go here now

File: recipes/yaehmop/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  license_file: license.txt
+  summary: 'Yet Another extended Hueckel Molecular Orbital Package'
+  doc_url: https://github.com/greglandrum/yaehmop/blob/master/docs/bind_manual.pdf
+  dev_url: https://github.com/greglandrum/yaehmop
+  
+extra:
+  recipe-maintainers:
+    - richardjgowers
+    - micaela-matta
+    - greglandrum
```
Comment (beckermr) [inline recipes/yaehmop/meta.yaml]: We need to make sure @greglandrum is ok being a maintainer? They should chime in here to confirm or deny!

Comment (richardjgowers) [reply recipes/yaehmop/meta.yaml]: @beckermr got the thumbs up on gh

**Pull request merged**